src/iface-ethernet-ip.c

```diff
@@ -206,6 +206,34 @@ static void attempt_connect(struct l_timeout *to, void *user_data)
 	l_timeout_modify(to, RECONNECT_TIMEOUT);
 }
 
+static int put_tag_name_created(struct knot_data_item *data_item)
+{
+	struct knot_data_item *data_item_aux;
+	int i;
+	int rc = -1;
```

Comment:
I don't understand the meaning of this number -1, I would recommend describing it.